apps/frontend/src/assets/css/variables.css

```diff
@@ -0,0 +1,182 @@
+.h1,
```

Comment:
Those are not variables tho
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascading_variables/Using_CSS_custom_properties